src/main/scala/no/ndla/learningpathapi/LearningpathApiProperties.scala

```diff
@@ -42,14 +42,17 @@ object LearningpathApiProperties extends LazyLogging {
   val ApiGatewayHost = "api-gateway.ndla-local"
   val ImageApiHost = propOrElse("IMAGE_API_HOST", "image-api.ndla-local")
   val InternalImageApiUrl = s"$ImageApiHost/image-api/v2/images"
+
   val NdlaFrontendHost = Environment match {
     case "prod"  => "ndla.no"
     case "local" => "localhost:30017"
     case _       => s"ndla-frontend.$Environment.api.ndla.no"
   }
-  val NdlaFrontendHostNames = Set("beta.ndla.no",
-                                  s"ndla-frontend.$Environment.api.ndla.no",
-                                  s"ndla-frontend.api.ndla.no")
+
+  val PublishRole = "learningpath:publish"
+
+  val NdlaFrontendHostNames =
+    Set("beta.ndla.no", s"ndla-frontend.$Environment.api.ndla.no", s"ndla-frontend.api.ndla.no")
```

Comment:
beta funker men brukes ikkje lenger. Kan plutselig være borte.

Reply:
Tror denne kun brukes for å ikke lagre url'en til embeds. 
Så lar den være en stund til om du ikke mener noe annet :)

Reply:
Ah, stemmer det. Brukes for å filtrere vekk hostnavnet fra en artikkel-embed-url. Kan være lurt å legge til den riktige i hvertfall. Altså ndla.no